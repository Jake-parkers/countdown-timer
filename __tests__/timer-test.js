import React from "react";
import Timer from "../components/timer";

import renderer from "react-test-renderer";

describe("<Timer/> test", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Timer
          minuteCounter={9}
          secondCounter={59}
          isTenSecsLeft={false}
          isTwentySecsLeft={false}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
