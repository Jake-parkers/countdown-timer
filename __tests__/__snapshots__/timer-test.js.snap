// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Timer/> blah blah 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "center",
      "marginTop": 50,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "borderRadius": 7,
        "height": "auto",
        "paddingBottom": 15,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#30374e",
          "fontSize": 50,
        }
      }
    >
      09
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 50,
        "marginTop": 15,
      }
    }
  >
     : 
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "borderRadius": 7,
        "height": "auto",
        "paddingBottom": 15,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#30374e",
          "fontSize": 50,
        }
      }
    >
      59
    </Text>
  </View>
</View>
`;

exports[`<Timer/> test renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "center",
      "marginTop": 50,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "borderRadius": 7,
        "height": "auto",
        "paddingBottom": 15,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#30374e",
          "fontSize": 50,
        }
      }
    >
      09
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 50,
        "marginTop": 15,
      }
    }
  >
     : 
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "borderRadius": 7,
        "height": "auto",
        "paddingBottom": 15,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#30374e",
          "fontSize": 50,
        }
      }
    >
      59
    </Text>
  </View>
</View>
`;
